inventory/sample/group_vars/k8s-cluster/k8s-net-calico.yml

```diff
@@ -23,6 +23,10 @@
 # Global as_num (/calico/bgp/v1/global/as_num)
 # global_as_num: "64512"
 
+# If doing peering with node-assigned asn where the globas does not match your nodes, you want this
```

Comment:
As discussed on slack, until the behaviour in the case is sorted out, this seems ok to me as it does not change the current behaviour